### ISSUE TITLE
Make it possible to include "unused" models in the generated swagger documentation

### ISSUE THREAD

**Issue by volfpeter** [python-restx/flask-restx#90]
Hi,

We use `flask-accepts` (`marshmallow`+`flask-restx`) to document and marshal our API. We would like to use the same tools to validate our custom JSON headers, but currently these extra models can't appear in the generated documentation because they are not a payload on any of our API endpoints.

Possible solution:

Add this bit of code to `Swagger.as_dict()` in `flask_restx/swagger.py`:
```Python
# should_add_unused_models could come from a RESTX_INCLUDE_ALL_MODELS env var, default is False
if should_add_unused_models:
    for m in self.api.models:
        self.register_model(m)
```

If you feel this request is reasonable, I would be happy to contribute a PR.

Thanks!

**Comment by volfpeter** [comment]
I've submitted the pull request that implements this feature. Could you please have a look at the issue and PR? Thanks!

**Comment by volfpeter** [comment]
Status: the feature is ready, I'll add the documentation for this feature once the rest of the `RESTX_` options are documented.

**Comment by ziirish** [comment]
Addressed in #104
